### ISSUE TITLE
Missing TSS scores with `--ignore-read-groups`

### ISSUE THREAD

**Issue by jennylsmith** [ParkerLab/ataqv#29]
Hello, 

I have started using ATAQV and its been a great help in QCing multiple samples - thanks for creating a great tool!  I noticed that I was lacking TSS scores when I ran ataqv with `--ignore-read-groups`. I am using ataqv version 1.3.0 and the help message doesn't indicate that this option results in no TSS enrichment score. The log files indicate that TSS enrichment score is being calculated even with `--ignore-read-groups` - but reports -nan as the value. 

I attached two log files for ataqv run with and without `--ignore-read-groups` option set. Any information about this would be appreciated as I am using ataqv in a data processing pipeline and I am hoping to avoid any segmentation fault
[ataqv_v5.log](https://github.com/ParkerLab/ataqv/files/10711761/ataqv_v5.log)
[ataqv_v6.log](https://github.com/ParkerLab/ataqv/files/10711762/ataqv_v6.log)
 errors that I'd seen on some of the other issues. 

```
  Quality Indicators
  ------------------
  Short to mononucleosomal ratio: 3.868
[ataqv_v5.log](https://github.com/ParkerLab/ataqv/files/10711735/ataqv_v5.log)
[ataqv_v6.log](https://github.com/ParkerLab/ataqv/files/10711736/ataqv_v6.log)

  High quality, nonduplicate, properly paired, uniquely mapped autosomal alignments: 52662996
    as a percentage of autosomal reads: 91.617%
    as a percentage of all reads: 64.191%
  TSS enrichment: -nan
```

**Comment by porchard** [comment]
Hi,

Thanks for passing this on. TSS enrichment should certainly still be calculated with `--ignore-read-groups`. Looking back at one of my recent ataqv jobs (v. 1.3.0, with option `--ignore-read-groups`) I see it was successfully output.

Would it be possible to share each of your input files?

**Comment by jennylsmith** [comment]
Hello, I just sent an email referencing this github issue that has a OneDrive link to the inputs. Please let me know if you didn't receive it or prefer another method of data sharing. Thanks for looking into this!

**Comment by porchard** [comment]
Hi,

Thanks for sending your files. This has been fixed in commit [1568623](https://github.com/ParkerLab/ataqv/commit/15686234e707c3d779a8e45f39a390c29a8691f5)

**Comment by jennylsmith** [comment]
Thank you @porchard!
